external/fv3fit/fv3fit/emulation/layers/fields.py

```diff
@@ -136,6 +136,7 @@ def __init__(
         sample_out: Optional[tf.Tensor] = None,
         denormalize: Optional[str] = None,
         enforce_positive: bool = False,
+        tendency_name: str = None,
```

Comment:
This is not a valid type hint. Type hints are pretty useful...I'm wondering if this part of the code is not being checked.
```suggestion
        tendency_name: Optional[str] = None,
```